.github/CONTRIBUTING.md

```diff
@@ -1,40 +1,45 @@
 # How to contribute
 
-We are really glad you are reading this contribution guide. This means you care about the quality of your contributions.
+We are really glad that you are reading this contribution guide.
+This means you care about the quality of your contributions.
 
 ## Submitting patches via Github
 
 ### Prerequisite for submitting patches/PR's
 
-1. Basic knowledge about how to us Git(hub)
-  * Properly setting up git with `git config`
+1. Basic knowledge about how to use Git(hub)
+  * [Properly](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup) setup your git client.
```

Comment:
“setup” is [not a verb](https://en.wiktionary.org/wiki/setup#Verb), it should be “set up”.